internal/scorecard/helpers/test_definitions.go

```diff
@@ -102,6 +103,17 @@ func (ts *TestSuite) TotalScore() (score int) {
 	return int(floatScore * (100 / addedWeights))
 }
 
+// Apply label selectors removing tests that do not match
```

Comment:
The first word should be the name of the function:
```suggestion
// ApplySelector removes tests from the TestSuite whose labels do not match selector.
```

Reply:
updated.